data-model/SDP-data-model-templates/place_sdp-template.json

```diff
@@ -1,9 +1,19 @@
 {
-  "id": "ARK/URI(?)",
-  "label": "",
+  "id": "URI",
   "type": "CV",
   "place_type": "CV",
```

Comment:
FYI I'm still on the fence about this field. I'm thinking it would have keywords like "settlement", "mountain", "region", river", "building", etc. I think this could be a useful data point to include, but I'm not convinced it's something we want users to worry about inputting. And I also don't know if it distracts from the importance of these records as name authorities?

Connecting a place record to an external authority could give us some of this information without having to input it ourselves. Still, I could go either way.